src/components/Avatar.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from "react";
+import { Css, Typography } from "src/Css";
+import { useTestIds } from "src/utils";
+
+export interface AvatarProps {
+  src: string | undefined;
+  name: string;
+  size?: AvatarSize;
+  showName?: boolean;
+}
+
+export function Avatar({ src, name, size = "md", showName = false, ...others }: AvatarProps) {
+  const tid = useTestIds(others, "avatar");
+  const px = sizeToPixel[size];
+  const [showFallback, setShowFallback] = useState(src === undefined);
+  const styles = Css.br100.wPx(px).hPx(px).overflowHidden.$;
+
+  const img = showFallback ? (
+    <div css={{ ...styles, ...Css[sizeToFallbackTypeScale[size]].bgLightBlue700.white.df.aic.jcc.$ }} {...tid}>
+      {nameToInitials(name)}
+    </div>
+  ) : (
+    <img
+      src={src}
+      alt={name}
+      css={{ ...styles, ...Css.objectCover.$ }}
+      onError={() => setShowFallback(true)}
```

Comment:
nit: Not a huge fan of making something like an `<Avatar />` stateful. Seems like there are other tools to handle fallbacks, like this:

https://stackoverflow.com/a/980910/12022691